src/ImageSharp/Image.WrapMemory.cs

```diff
@@ -403,6 +403,7 @@ public abstract partial class Image
     /// <typeparam name="TPixel">The pixel type</typeparam>
     /// <param name="configuration">The <see cref="Configuration"/></param>
     /// <param name="pointer">The pointer to the target memory buffer to wrap.</param>
+    /// <param name="byteLength">The byte length of the memory allocated.</param>
```

Comment:
You're gonna hate me but I think `bufferSizeInBytes` is a better parameter name as it's more consistant with our existing memory allocator APIs